### ISSUE TITLE
Tenant License Portability?

### ISSUE THREAD

**Issue by Mike-Crowley** [MicrosoftDocs/microsoft-365-docs#12478]
In the multiple tenants and ongoing maintenance sections, It would be helpful to provide some resource for moving licenses between tenants. Even if the answer is "contact sales" (as implied here: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10465), it would be helpful to have official guidance. This is especially true if an organization wants to migrate users without doubling their license cost for the duration of each migration.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 8e4409fa-932a-7b1c-7da0-b35836e48048
* Version Independent ID: bf893c77-d4e1-d148-07bd-51a99228f17d
* Content: [Step 1. Your Microsoft 365 for enterprise tenants](https://learn.microsoft.com/en-us/microsoft-365/solutions/tenant-management-tenants?view=o365-worldwide)
* Content Source: [microsoft-365/solutions/tenant-management-tenants.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/solutions/tenant-management-tenants.md)
* Service: **o365-solutions**
* GitHub Login: @kelleyvice-msft
* Microsoft Alias: **kvice**

**Comment by Mike-Crowley** [comment]
Thanks, does this mean there is no process to move licenses between tenants for commercial (non enterprise agreement) customers?

**Comment by msbemba** [comment]
Yes, the option is to cancel and purchase licenses.

**Comment by msbemba** [comment]
@yogkumgit  This can be closed.

**Comment by yogkumgit** [comment]
@msbemba Is the PR still valid?

**Comment by msbemba** [comment]
@yogkumgit Yes, the PR is valid. We can leave this open till the PR is merged. Sorry about that.

**Comment by officedocspr5** [comment]
To make it easier for you to submit feedback on articles on learn.microsoft.com, we're transitioning our feedback system from GitHub Issues to a new experience. 

As part of the transition, this GitHub Issue will be moved to a private repository. We're moving Issues to another repository so we can continue working on Issues that were open at the time of the transition. When this Issue is moved, you'll no longer be able to access it.

If you want to provide additional information before this Issue is moved, please update this Issue before December 15th, 2023.

With the new experience, you no longer need to sign in to GitHub to enter and submit your feedback. Instead, you can choose directly on each article's page whether the article was helpful. Then you can then choose one or more reasons for your feedback and optionally provide additional context before you select **Submit**.

Here's what the new experience looks like.

**Note**: The new experience is being rolled out across learn.microsoft.com in phases. If you don't see the new experience on an article, please check back later.

First, select whether the article was helpful:

![Image showing a dialog asking if the article was helpful with yes and no answers.](https://learn.microsoft.com/Office/media/internal/new-feedback-yes-no.jpg)

Then, choose at least one reason for your feedback and optionally provide additional details about your feedback:

| Article was helpful | Article was unhelpful |
|:--|:---|
|![Image showing a dialog asking how the article was helpful with several options.](https://learn.microsoft.com/Office/media/internal/new-feedback-helpful.jpg) | ![Image showing a dialog asking how the article wasn't helpful with several options.](https://learn.microsoft.com/Office/media/internal/new-feedback-unhelpful.jpg)|

Finally, select **Submit** and you're done!
